lib/Form/Checkbox/Group.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from './';
+
+export default class Group extends Component {
+  static propTypes = {
+    choices: PropTypes.arrayOf(PropTypes.shape({
+
+    })).isRequired,
+    onChangeHandler: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    const selected = props.choices.filter(c => c.checked);
+    this.state = { choices: props.choices, selected };
+    this.onChangeHandler = this.onChangeHandler.bind(this);
+    this.choiceIsAlreadySelected = this.choiceIsAlreadySelected.bind(this);
+    this.removeChoice = this.removeChoice.bind(this);
+    this.addChoice = this.addChoice.bind(this);
+  }
+
+  onChangeHandler(e) {
+    const choice = this.state.choices.find(c => c.id === e.target.id);
+
+    const selected = this.choiceIsAlreadySelected(choice) ? this.removeChoice(choice) : this.addChoice(choice);
+    this.setState({ selected });
+
+    this.props.onChangeHandler(selected);
+  }
+
+  choiceIsAlreadySelected(choice) {
+    return this.state.selected.findIndex(c => c === choice) >= 0;
```

Comment:
`findIndex` isn't support in IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

Maybe use `filter` instead and look for the `id` within the selected objects?

Reply:
Thanks, have done now :+1: